src/pages/configure/configure.jsx

```diff
@@ -97,6 +98,9 @@ const Configure = () => {
 					<h2>Eventing Config</h2>
 					<div className="divider" />
 					<EventingConfigure dbType={eventing.dbType} dbList={dbList} col={eventing.col} handleSubmit={handleEventingConfig} />
+					<h2>Export/Import Project Config</h2>
+					<div className="divider" />
+					<ExportImport object={projects[0]} />
```

Comment:
The `ExportImport` component shouldn't be backend or redux aware. The `ExportImport` component should just take two props - 
- `projectConfig`: current project config in JSON (shouldn't be hardcoded as `projects[0]`coz it's not gauranteed that the first project will be the selected project)
- `setProjectConfig`: A func that takes json object, puts it to backend, on success changes redux and triggers notify depending on context